sql/core/src/main/scala/org/apache/spark/sql/streaming/ui/StreamingQueryPage.scala

```diff
@@ -206,12 +206,13 @@ class StreamingQueryPagedTable(
   }
 }
 
-case class StructuredStreamingRow(
+private[ui] case class StructuredStreamingRow(
     duration: Long,
     avgInput: Double,
     avgProcess: Double,
     streamingUIData: StreamingQueryUIData)
 
+private[ui]
```

Comment:
nit:
shall we make it 
```
private[ui] class StreamingQueryDataSource(
    uiData: Seq[StreamingQueryUIData],
    ...
```

Reply:
Make sense, will fix in the next commit.